common/build.gradle

```diff
@@ -28,7 +28,7 @@ codeCoverageReport {
     coverage.enabled = enableCodeCoverage
 }
 
-def common4jVersion = "0.0.+"
+def common4jVersion = "1.0.8-CBATest"
```

Comment:
I believe this is for your feature branch only

Reply:
yep this was for a private preview build. I'll change this along with the versions.properties variables back when my feature branch is ready to be merged into dev